src/OrbitGl/QtTextRenderer.cpp

```diff
@@ -110,8 +100,11 @@ void QtTextRenderer::AddText(const char* text, float x, float y, float z, TextFo
       formatting.max_size == -1.f ? FLT_MAX : viewport_->WorldToScreen({formatting.max_size, 0})[0];
   // Find out how many characters from text can fit into max_width via a binary search.
   std::string text_as_string(text);
-  size_t idx_min = 0;
+  size_t idx_min = 1;
   size_t idx_max = text_length;
+  if (GetStringWidth(text_as_string.substr(0, 1).c_str(), formatting.font_size) > max_width) {
+    return;
```

Comment:
We are not updating pos and size in this return. Can you add a TODO/bug at least?

Reply:
Yeah, same as with the return in the beginning ... changed to zeroing out the pos and size in the beginning.